main.go

```diff
@@ -21,23 +21,18 @@
 package main
 
 import (
+	"github.com/Azure/azure-storage-azcopy/v10/cmd"
+	"github.com/Azure/azure-storage-azcopy/v10/common"
 	"log"
-	"math/rand"
 	"os"
 	"path"
 	"runtime"
-	"time"
-
-	"github.com/Azure/azure-storage-azcopy/v10/cmd"
-	"github.com/Azure/azure-storage-azcopy/v10/common"
 )
 
 // get the lifecycle manager to print messages
 var glcm = common.GetLifecycleMgr()
 
 func main() {
-	rand.Seed(time.Now().UnixNano()) // make sure our random numbers actually are random (but remember, use crypto/rand for anything where strong/reliable randomness is required
```

Comment:
What is the reasoning behind removing this randomization? /gen

Reply:
This is no longer necessary with the latest Go version. Now rand is by default generating a random seed.